src/main/scala/beam/sim/BeamMobsim.scala

```diff
@@ -314,8 +314,13 @@ class BeamMobsimIteration(
   }
 
   private def scheduleRideHailManagerTimerMessages(): Unit = {
-    if (config.agents.rideHail.repositioningManager.timeout > 0)
-      scheduler ! ScheduleTrigger(RideHailRepositioningTrigger(0), rideHailManager)
+    if (config.agents.rideHail.repositioningManager.timeout > 0) {
```

Comment:
@colinsheppard @wrashid do you think this comment is enough to understand why it is done in this way?

Reply:
that's fine... I will eventually solve this so that stagger is no longer needed but this will do for now